### DISCUSSION TITLE
Writing a value to a non-existent node

### DISCUSSION THREAD

**Discussion by unregistered33** [FreeOpcUa/opcua-asyncio#1305] [General]
Hi!
How to check if node exists on the server?
Thanks!

**Comment by oroulet** [comment]
try to read a paramter for example. Any read (or write) operation anyway end up with one network call, with about the same cost

**Comment by AndreasHeine** [comment]
after the client has connected you browse/read the nodes for the first time and after that you subscribe the modelchangeevents https://reference.opcfoundation.org/Core/Part3/9.32.7/

as long these events are not been emitted the node should be still present!
